pocketlint/formatcheck.py

```diff
@@ -939,6 +945,7 @@ def check_lines(self):
         for line_no, line in enumerate(self.lines):
             line_no += 1
             self.check_length(line_no, line)
+            self.check_semantic_newline()
```

Comment:
this is fine, but don't forget to pass the `line_no, line` arguments to this check.

line_no is important so that the error report will pin-point the place with the error